internal/pithosctl/pkg/pithos/healthz.go

```diff
@@ -142,6 +142,12 @@ func (h *Healthz) createObject() error {
 }
 
 func (h *Healthz) GetObject() error {
-	_, err := h.Client.GetObject(h.Bucket, objectName, minio.GetObjectOptions{})
+	reader, err := h.Client.GetObject(h.Bucket, objectName, minio.GetObjectOptions{})
```

Comment:
The `reader` is `minio.Object` and `GetObject` creates a bunch of channels and starts a goroutine. If you don't call `reader.Close` these are all going to be left behind. Maybe not so drastic with channel, but goroutines will pile up.

Reply:
I swear me put defer close in the code. Not sure how did I delete it and didn't push. 
Anyway, addressed in the next commit.